scopes/harmony/application/run.cmd.ts

```diff
@@ -62,5 +62,22 @@ export class RunCmd implements Command {
     if (isOldApi) {
       this.logger.console(`${appName} app is running on http://localhost:${port}`);
     }
+
+    /**
+     * normally, when running "bit run <app-name>", the app is running in the background, which keeps the event loop busy.
+     * when the even loop is buys, the process doesn't exit, which is what we're looking for.
```

Comment:
typo here
It should be "when the event loop is busy" instead of "when the even loop is buys"